keras/src/callbacks/orbax_checkpoint.py

```diff
@@ -24,6 +23,13 @@
     pass
 
 
+def _get_orbax_multihost():
+    """Get the orbax multihost module with lazy import."""
+    import orbax.checkpoint as ocp
+
+    return ocp.multihost
```

Comment:
Can we use the "normal" LazyModule approach? Can it piggyback on the existing `keras.src.utils.module_utils.ocp`? Or do we need a separate one?